src/utils/dif_upload.rs

```diff
@@ -453,7 +453,9 @@ fn try_open_zip<P>(path: P) -> Result<Option<ZipFileArchive>, Error>
 where
     P: AsRef<Path>,
 {
-    if path.as_ref().extension() != Some("zip".as_ref()) {
+    let ext = path.as_ref().extension();
+    // Also look into Android Archives for native symbols
+    if ext != Some("zip".as_ref()) && ext != Some("aar".as_ref()) {
```

Comment:
`apk` and `aab` would also be possible, but I guess all of them (including `aar`) already have stripped debug symbols, not sure if it's any useful, have you tried @bruno-garcia ?

Reply:
I tried but not frames from the sentry-native SDK were in the stacktrace so not sure if line numbers were available.
It's likely best to revert this since we're uploading symbols before AGP strips symbols.

Good catch @marandaneto